examples/minimal-mdns/advertiser.cpp

```diff
@@ -61,6 +64,8 @@ constexpr uint16_t kOptionCommisioningShordDiscriminator = 's';
 constexpr uint16_t kOptionCommisioningLongDiscriminaotr  = 'l';
 constexpr uint16_t kOptionCommisioningVendorId           = 0x100; // v is used by 'version'
 constexpr uint16_t kOptionCommisioningProductId          = 'p';
+constexpr uint16_t kOptionCommisioningPairingInstr       = 'I';
+constexpr uint16_t kOptionCommisioningPairingHint        = 'H';
```

Comment:
using uppercase seems a bit different than existing flags. I imagine 'H' is to not conflict with 'h'. I would either then update vendorId to use 'V' (and maybe make all others uppercase too) or make the conflicting ones only have a long version like '--pairing-hint'.

Reply:
I think sticking to the long version for conflicting ones is more readable. Will do that.